FlowCryptUI/Nodes/AttachmentNode.swift

```diff
@@ -43,6 +43,7 @@ public final class AttachmentNode: CellNode {
         imageNode.tintColor = .gray
         buttonNode.tintColor = .gray
         deleteButtonNode.setImage(UIImage(named: "cancel")?.tinted(.gray), for: .normal)
+        deleteButtonNode.accessibilityIdentifier = "deleteAttachmentButton"
```

Comment:
Also here if possible to enumerate them? So this would be `deleteAttachmentButton1` maybe.

Reply:
Will add enumeration here too